src/content/blog/web-framework-performance-report.mdx

```diff
@@ -0,0 +1,132 @@
+
+Across all tracked websites, the median performance score was a 34/100. To that end, half of our tested frameworks (Astro, SvelteKit, and Remix) did come in above the internet average.
+
+<LighthousePercentileChart />
+
+By breaking the data down by percentile, we can start to see some slightly more encouraging numbers with Astro and SvelteKit reaching a score of 90+ in the p90 or p95 percentiles. However, the data clearly shows that all websites and frameworks (including Astro) still struggle to achieve good performance in real-world situations.
+
+# **The Cost of JavaScript**
+
+The last thing that we wanted to explore was the relationship between framework choice, performance, and total JavaScript payload size in real-world usage. Do the fastest frameworks tend to be the ones that also send the least amount of JavaScript to the client?
+
+<CostOfJSChart />
+
+The trend in the data is clear: sites that ship less JavaScript tend to perform better. However, there are too many factors at play for us to confidently tie this trend back to the choice of web framework itself. It could be the case that some frameworks encourage/discourage JavaScript differently than others, but more research is needed before we draw any conclusions.
+
+# Methodology & Limitations
+
+This report was compiled from several publicly available datasets. You can learn more about these datasets and their methodology here: [HTTP Archive methodology](https://almanac.httparchive.org/en/2021/methodology#dataset), [CrUX methodology](https://developer.chrome.com/docs/crux/methodology/), and [CWV Technology Report methodology](https://discuss.httparchive.org/t/new-dashboard-the-core-web-vitals-technology-report/2178).
+
+Due to capacity limitations, our analysis only looks at the homepages of every tracked website. One benefit of this limitation is that there is less variance in the purpose and use-case of each analyzed website. However, on drawback is that this also means interior pages (like `/about` and `/admin/...` pages) and the technologies that they use are not analyzed and therefore excluded from our analysis.
+
+Another limitation that is unexplored in this report is the impact of a framework's age on measured web performance. The older frameworks measured here (Gatsby, Next.js, Nuxt) have a longer tail of legacy websites running older versions of their framework which are included in the dataset. This creates a situation where only the newer frameworks (Astro, Remix, SvelteKit) can be assumed to be running more modern versions of their software from the last 1-2 years, which could impact the data. This is a limitation of the data that we have available, but it is something that we hope to explore in future reports.
```

Comment:
Is it worth moving methodology up to the top? These seem like good things to keep in mind _while_ reading the data, not _after_ reading the data.

Reply:
I link to it in the "data" section, but it's a pretty big section to include at the top. The fear is the opposite and that it's intimidating, getting in the way of the reader trying to see the data. Let me see about making the link more prevalent.